<ui version="4.0" >
 <class>GridSettings</class>
 <widget class="QWidget" name="GridSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Grid</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="segmentsWidth" >
        <property name="maximum" >
         <number>10000</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Width-Segments:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Height-Segments:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="height" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>999999.989999999990687</double>
        </property>
        <property name="minimum" >
         <double>0.001000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>4.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QPushButton" name="insertButton" >
        <property name="text" >
         <string>Insert Grid</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="segmentsHeight" >
        <property name="maximum" >
         <number>10000</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="width" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>1000000.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>0.001000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>4.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>width</tabstop>
  <tabstop>height</tabstop>
  <tabstop>segmentsWidth</tabstop>
  <tabstop>segmentsHeight</tabstop>
  <tabstop>insertButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
